docs/safetensors.md

```diff
@@ -0,0 +1,136 @@
+# SafeTensors Reader User Guide
+
+The SafeTensors reader in VirtualiZarr allows you to reference tensors stored in SafeTensors files. This guide explains how to use the reader effectively.
+
+## What is SafeTensors Format?
+
+SafeTensors is a file format for storing tensors (multidimensional arrays) that offers several advantages:
+- Safe: No use of pickle, eliminating security concerns
+- Efficient: Zero-copy access for fast loading
+- Simple: Straightforward binary format with JSON header
+- Language-agnostic: Available across Python, Rust, C++, and JavaScript
+
+The format consists of:
+- 8 bytes (header size): little-endian uint64 containing the size of the header
+- JSON header: Contains metadata for all tensors (shapes, dtypes, offsets)
+- Binary data: Contiguous tensor data
+
+## How VirtualiZarr's SafeTensors Reader Works
+
+VirtualiZarr's SafeTensors reader allows you to:
+- Work with the tensors as xarray DataArrays with named dimensions
+- Access specific slices of tensors from cloud storage
+- Preserve metadata from the original SafeTensors file
+
+## Basic Usage
+
+Opening a SafeTensors file is straightforward:
+
+```python
+import virtualizarr as vz
+
+# Open a SafeTensors file
+vds = vz.open_virtual_dataset("model.safetensors")
+
+# Access tensors as xarray variables
+weight = vds["weight"]
+bias = vds["bias"]
+
+# Convert to numpy arrays when needed
+weight_array = weight.values
+bias_array = bias.values
+```
+
+## Custom Dimension Names
+
+By default, dimensions are named generically (e.g., "weight_dim_0", "weight_dim_1"). You can provide custom dimension names for better semantics:
+
+```python
+# Define custom dimension names
+custom_dims = {
+    "weight": ["input_dims", "output_dims"],
+    "bias": ["output_dims"]
+}
```

Comment:
This is neat, and I'm totally fine keeping it, but FYI an alternative would be to not have this kwarg and encourage the use of xarray's `.rename_dims()` method instead.

Though there is no .rename method on the `ManifestStore`, whereas your approach would allow you to choose dimension names on the `ManifestStore`.